integration/commands_administration_compat_test.go

```diff
@@ -164,7 +164,7 @@ func TestCommandsAdministrationCompatCollStatsCappedCollection(t *testing.T) {
 
 			// TODO https://github.com/FerretDB/FerretDB/issues/3582
 			assert.EqualValues(t, tc.expectedSize, must.NotFail(targetDoc.Get("maxSize")))
-			assert.EqualValues(t, must.NotFail(compatDoc.Get("maxSize")), must.NotFail(targetDoc.Get("maxSize")))
+			assert.LessOrEqual(t, int64(must.NotFail(compatDoc.Get("maxSize")).(int32)), must.NotFail(targetDoc.Get("maxSize")))
```

Comment:
Why do we use `LessOrEqual` here?

Reply:
We're comparing maxSize disk space for Mongo and FerretDB, which is unlikely the same.
I thought about removing the assertion but then thought maybe leaving it there for having a reference might give us a hint if something goes wrong. Need to check it thoroughly again.

Reply:
I suggest checking it again, yes. `maxSize` is set when a capped collection is created, it's an exact number, not an estimation. So, I don't see reasons why they could be different :) There are some changes in MongoDB 7 regarding the size: https://www.mongodb.com/docs/rapid/release-notes/7.1-compatibility/#:~:text=Starting%20in%20MongoDB%207.1%2C%20MongoDB,with%20earlier%20versions%20of%20MongoDB. With these changes, we might want to ask ourselves if we want to support a similar logic in FerretDB :)

Reply:
You're right. I've changed the application code to handle that (it was two small changes and it seemed the right thing to do), reverted to the previous check, and updated the expected values.